[FIX] report_aeroo: Fix parser and old lib.

nicomacr opened the pull request.

nicomacr committed:
[FIX] report_aeroo: Fix parser and old lib.

roboadhoc commented:
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/aeroo_reports/pull/41).

jjscarafia committed:
[MIG] report_aeroo_sample: Migration to 15.0

jjscarafia committed:
[REM] unneeded encodings

jjscarafia committed:
[REM] unused wizards and reorganice files

jjscarafia committed:
[REVERT] revert and fix dont use variables on self
Revertimos este commit nuestro https://github.com/ingadhoc/aeroo_reports/commit/5d4f48a7a077dee0275b24463814171f20897157
Y aplicamos nueva forma para adaptarnos a este cambio de odoo https://github.com/odoo/odoo/commit/ea3e39506a2b360545ec0cd1d3235e69ad16ea6c
Ya no es posible guardar variables en "self", hacemos el cambio rapido a guardar cosas en "self.env"
Eventualmente, si esto no funciona por alguna razón, se podria ver de usar el context, pero el context es un frozenddict y habria que ver cual es la forma prolija.
Si no, en el mismo commit de odoo hay algunos ejemplos de como odoo lo cambio en otros casos (con un campo y store=False por ejemplo)

jjscarafia committed:
[IMP] update genshi to avoid log warning

jjscarafia committed:
[FIX] print report with custom context

jjscarafia committed:
[FIX] parser load from file
Borramos usabilidad compleja de parser que no usamos. Basicamente dejamos:
a) por defecto parser nativo
b) se puede poner nombre de otro modelo utilizado como parser

jjscarafia committed:
fixup

jjscarafia committed:
[REF] clean code

nicomacr commented:
@roboadhoc r+ override=ci/oba_without_tests

roboadhoc commented:
@nicomacr, you may want to rebuild or fix this PR as it has failed CI.

nicomacr commented:
@roboadhoc rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward
